edit-post/api/index.js

```diff
@@ -1,3 +1,7 @@
+export {
+	registerMoreMenuItem,
+	getMoreMenuItems,
```

Comment:
`getMoreMenuItems ` - does it need to be exposed? Is there a valid use case for it? The name is also confusing because it contains only the items registered in the plugins section.